sonar-java-plugin/src/main/java/org/sonar/plugins/java/CommonRulesSonarWayProfile.java

```diff
@@ -39,13 +39,16 @@ public CommonRulesSonarWayProfile(XMLProfileParser parser) {
 
   @Override
   public RulesProfile createProfile(ValidationMessages validationMessages) {
-    InputStream input = getClass().getResourceAsStream("/org/sonar/plugins/java/common_rules_sonar_way.xml");
-    InputStreamReader reader = new InputStreamReader(input, Charsets.UTF_8);
-    try {
-      return parser.parse(reader, validationMessages);
-    } finally {
-      IOUtils.closeQuietly(reader);
+    RulesProfile rulesProfile = null;
+    
+    try (InputStream input = getClass().getResourceAsStream("/org/sonar/plugins/java/common_rules_sonar_way.xml");
+      InputStreamReader reader = new InputStreamReader(input, Charsets.UTF_8)) {
+      rulesProfile = parser.parse(reader, validationMessages);
+    } catch (IOException e) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)